backends/concrete-cuda/implementation/src/bootstrap_amortized.cu

```diff
@@ -85,6 +86,8 @@ void scratch_cuda_bootstrap_amortized_32(void *v_stream, uint32_t gpu_index,
         input_lwe_ciphertext_count, max_shared_memory, allocate_gpu_memory);
     break;
   default:
```

Comment:
@BourgerieQuentin as mentioned in one of the weekly's the problem when I had the bug was that the optimizer was using the CPU search space because of a flag that I had switched in the compiler engine, so not an issue with the optiizer. The problem was that because those switches don't have defaults, and silently fall through to deallocate pointers that were never allocated for example, when there's a bug outside of that code the behaviour is difficult to investigate.

Reply:
So it's fixed?

Reply:
The bug is fixed indeed. If you prefer that I don't add the defaults and errors in switches, I can remove.

Reply:
that's good :pray: